apps/api-frontend/src/app/pages/info-page/info.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './info.css';
+
+export default function InfoPage() {
+  return (
+    <div className="info">
+      <h3>Info about Schopal</h3>
+      <p>
+        Schopal is a search engine for Scholarly articles. Using open source APIs, Schopal finds scholarly articles, and then analyzes them according to your search.
+        Schopal will then return results <div className="bold">by paragraph</div> rather than by article. Thus, you may see the same article appear twice.
```

Comment:
Why not use `<b>text</b>` instead of `<div className="bold">text</div>`?

Reply:
+1 good catch Casey